src/catalog/src/durable/upgrade/v69_to_v70.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::sync::LazyLock;
+
+use mz_pgrepr::oid::NETWORK_POLICIES_DEFAULT_POLICY_OID;
+use mz_sql::plan::{NetworkPolicyRuleAction, NetworkPolicyRuleDirection};
```

Comment:
Pretty strong preference to define the default network policy rules with the protos themselves as opposed to imported Rust types. It's just a lot easier to reason about a migration if it has no imports other than the protobuf types

Reply:
That makes sense, and it's much cleaner code